From 025493565754a8e955c4a72b56cc9892d292ea89 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@rvc.renesas.com>
Date: Wed, 20 Feb 2019 14:52:35 +0700
Subject: [PATCH 5/9] Network-video-wireless-streaming: Remove unavailable
 properties

Environment of RCar Gen3 doesn't have these properties, remove them.
Also fix "OR" condition when checking parameters

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@rvc.renesas.com>
---
 RZ_GST-network-demo_video-wireless-streaming.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/RZ_GST-network-demo_video-wireless-streaming.sh b/RZ_GST-network-demo_video-wireless-streaming.sh
index 2ecef8e..e1dcb20 100755
--- a/RZ_GST-network-demo_video-wireless-streaming.sh
+++ b/RZ_GST-network-demo_video-wireless-streaming.sh
@@ -22,7 +22,7 @@ function f_error()
 	f_help
 }
 
-if [ "$1" = "--help" ] | [ -z "$1" ]; then
+if [ "$1" = "--help" ] || [ -z "$1" ]; then
 	f_help
 else
 	test-multicast2 &
@@ -30,10 +30,11 @@ else
 	gst-launch-1.0 -e \
 	filesrc location=$1 ! \
 	qtdemux ! \
-	omxh264dec no-copy=true ! \
-	vspmfilter outbuf-alloc=true ! \
-	video/x-raw,format=NV12,width=640,height=480 ! \
-	omxh264enc target-bitrate=5242880 num-p-frames=0 ! \
+	h264parse ! \
+	omxh264dec ! \
+	vspfilter ! \
+	video/x-raw,width=640,height=480 ! \
+	omxh264enc target-bitrate=5242880 ! \
 	h264parse ! \
 	video/x-h264,stream-format=avc,alignment=au ! \
 	rtph264pay pt=96 name=pay0 config-interval=3 ! \
-- 
2.7.4

